src/containerapp/azext_containerapp/tests/latest/test_containerapp_create_update_with_source.py

```diff
@@ -32,14 +32,6 @@ def test_containerapp_create_source_with_buildpack_e2e(self, resource_group):
         target_port = '8080'
         create_and_verify_containerapp_create_and_update(self, resource_group=resource_group, source_path=source_path, ingress=ingress, target_port=target_port)
 
-    @live_only()
-    @ResourceGroupPreparer(location="eastus")
-    def test_containerapp_create_artifact_with_buildpack_e2e(self, resource_group):

```

Comment:
Why need to remove test_containerapp_create_artifact_with_buildpack_e2e and test_containerapp_up_artifact_with_bullseye_buildpack_e2e?

Reply:
I have moved Java related test cases to [test_containerapp_create_update_up_java.py](https://github.com/Azure/azure-cli-extensions/blob/01c6aa6c9da7bf9c1606511a68a87fc9862baf66/src/containerapp/azext_containerapp/tests/latest/test_containerapp_create_update_up_java.py) and changed test method name, so the old ones are removed.